cantools/subparsers/plot.py

```diff
@@ -51,14 +51,23 @@
 try:
     from matplotlib import pyplot as plt
 except ImportError:
-    print("matplotlib package not installed. Required for producing plots.")
+    plt = None
 
 from .. import database
+from .. import errors
 
 
-plt.rcParams["date.autoformatter.hour"] = "%H:%M"
-plt.rcParams["date.autoformatter.minute"] = "%H:%M"
-plt.rcParams["date.autoformatter.microsecond"] = "%H:%M:%S.%f"
+class MatplotlibNotInstalledError(errors.Error):
+    def __init__(self):
+        super().__init__(
+            "The matplotlib package not installed and is required for producing plots.",
+        )
```

Comment:
![image](https://user-images.githubusercontent.com/543719/107388789-2f062c00-6ac4-11eb-8cfc-36830731ae5e.png)

When suggesting specific code changes it is simplest to do so with a 'suggestion'.  You start by highlighting the relevant lines when creating your comment (as you did) and then follow by clicking the +/- suggestoin better generally to insert the code such that you can edit it.  This makes it easy to integrate your already-written change instead of trying to recreate it in a local checkout etc etc.

```suggestion
        super().__init__("The matplotlib package not installed and is required "
                         "for producing plots.")
```

Reply:
ok